CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+If you're interested in contributing to this project,
+please start with the
+[general PDC contribution guidelines](https://github.com/PhilanthropyDataCommons/.github/blob/main/CONTRIBUTING.md).
+
+## Guidelines specific to this project
+
+### Populating "Recent Changes"
+
+If your change affects the user experience,
```

Comment:
You seem to be wrapping at ~40 characters which is fairly aggressive.

I think an 80 character wrap might be nicer?  It looks like the main CONTRIBUTING.md uses 72.

(I personally think wrapping in markdown is a losing battle but I know I'm in the minority here)

Reply:
I've adopted @jasonaowen's manner of semantic line breaks. Despite the semblance of a [spec](https://sembr.org/), IMO the core philosophy is to reduce each line to a discrete phrase that could be changed with minimal diff confusion on the surrounding text.

That said, maybe it's a losing battle and diff engines are good enough at highlighting mid-line changes.

If this is a strong objection I'll defer!

Reply:
I do think that when using a text editor (as opposed to reading in a web browser that flattens the whitespace) that approach becomes harder to read -- though I guess it's just more like reading poetry, where you can truly appreciate the WORDS being used and the shape of the prose...  that counts for *something*!

Honestly, whatever floats yinz's boats is fine (but please note that my approval won't stop be from making *fun* of the approach!)

Reply:
Grokked and I see your point!

However, in the interest of velocity (and with your overall approval), I'll save this discussion for another day.